run-local.sh

```diff
@@ -32,9 +32,32 @@ export HMPPS_AUTH_URL=https://sign-in-dev.hmpps.service.justice.gov.uk/auth
 # Make the connection without specifying the sslmode=verify-full requirement
 export SPRING_DATASOURCE_URL='jdbc:postgresql://${DB_SERVER}/${DB_NAME}'
 
-# Bring up the licences-db container needed for the application
-docker compose down --remove-orphans && docker compose pull
+# Bring up the front end containers needed for the application
+cd ../create-and-vary-a-licence
+echo "Front end setup ..."
+echo "Bringing down current containers ..."
+docker compose down
+echo "Installing npm ..."
+npm install
+echo "Pulling front end containers ..."
+docker compose -f docker-compose-dev.yml pull
+docker compose -f docker-compose-dev.yml up -d
+echo "Waiting for front end containers to be ready ..."
+until [ "`docker inspect -f {{.State.Health.Status}} gotenberg`" == "healthy" ]; do
```

Comment:
rather than having the details about frontend dependencies in this project, could we create a run-local.sh script in the frontend? 

and maybe have a `run-full.sh` in this project that calls both run-local.sh scripts?